MAUI/DataGrid/conditional-styling.md

```diff
@@ -0,0 +1,168 @@
+<ContentPage xmlns:syncfusion="http://schemas.syncfusion.com/maui">
+    <ContentPage.Resources>
+        <local:ColorConverter x:Key="converter"/>
+        <Style TargetType="syncfusion:DataGridRow">
+            <Setter Property="Background" Value="{Binding Converter={StaticResource converter}}" />
+        </Style>
+    </ContentPage.Resources>
+</ContentPage>
+{% endhighlight %}
+{% highlight xaml tabtitle="ColorConverter.cs"%}
+public class ColorConverter : IValueConverter
+{
+    object IValueConverter.Convert(object value, Type targetType, object parameter, CultureInfo info)
+    {
+        var input = (value as OrderInfo).OrderID;
+        if (input < 1003)
+            return Colors.Bisque;
+        else if (input < 1007)
+            return Colors.LightBlue;
+        else
+            return Colors.White;
+    }
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        throw new NotImplementedException();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Conditional DataGridRow styling in .NET MAUI DataGrid](Images/conditional-styling/maui-datagrid-conditional-datagridrowstyle.png)
+
+## How to style a particular row based on row index
+Styling can be applied to a particular row based on RowIndex property by writing the style for [DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html) TargetType.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml"%}
+<ContentPage xmlns:syncfusion="http://schemas.syncfusion.com/maui">
+    <ContentPage.Resources>
+        <local:ColorConverter x:Key="converter"/>
+        <Style TargetType="syncfusion:DataGridRow">
+            <Setter Property="Background" Value="{Binding Source={RelativeSource Mode=Self}, Converter={StaticResource Key=converter}}"/>
+        </Style>
+    </ContentPage.Resources>
+</ContentPage>
+{% endhighlight %}
+{% highlight xaml tabtitle="ColorConverter.cs"%}
+public class ColorConverter : IValueConverter
+{
+    object IValueConverter.Convert(object value, Type targetType, object parameter, CultureInfo info)
+    {
+        var dataGridRow = value as DataGridRow;
+        var rowIndex = dataGridRow.DataRow.RowIndex;
+        if (rowIndex == 3)
+           return Colors.LightBlue;
+        else
+            return Colors.White;
+    }
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        throw new NotImplementedException();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Conditional DataGridRow styling Based on RowIndex in .NET MAUI DataGrid](Images/conditional-styling/maui-datagrid-conditional-datagridrowstyle_basedonrowindex.png)
+
+## How to style a particular row based on RowData
+Styling can be applied to a particular row based on RowData property by writing the style for [DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html) TargetType.
```

Comment:
Styling can be applied to a particular row based on the RowData property by writing the style for the [DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html) TargetType.